tests/sidetrail/working/patches/cbc.patch

```diff
@@ -74,15 +74,15 @@ index 401ab760..f39cc7e2 100644
 +    //}
  
      /* Check the maximum amount that could theoretically be padding */
-     int check = MIN(255, (payload_and_padding_size - 1));
+     uint32_t check = MIN(255, (payload_and_padding_size - 1));
  
-     int cutoff = check - padding_length;
--    for (uint32_t i = 0, j = decrypted->size - 1 - check; i < check && j < decrypted->size; i++, j++) {
+     uint32_t cutoff = check - padding_length;
+-    for (size_t i = 0, j = decrypted->size - 1 - check; i < check && j < decrypted->size; i++, j++) {
 -        uint8_t mask = ~(0xff << ((i >= cutoff) * 8));
 -        mismatches |= (decrypted->data[j] ^ padding_length) & mask;
 -    }
```

Comment:
I don't understand why these three lines are marked as changed. Is it a white space change?

Reply:
The only change is in the `for` loop on line 80. Since this is a patch, it contains diff listings. Basically, the file is exactly what we see in a `git diff`.